src/MICore/Logger.cs

```diff
@@ -15,10 +15,15 @@
 
 namespace MICore
 {
+    public interface ILogger
+    {
+        void WriteLine(string line);
+    }
```

Comment:
>} [](http://example.com/codeflow?start=4&length=1)

Also add `public void WriteLine(string format, params object[] args)` since, in general, we should probably perfer code to call that overload?